editoast/src/models/fixtures.rs

```diff
@@ -422,3 +424,9 @@ pub async fn create_work_schedules_fixture_set(
 
     (work_schedule_group, work_schedules)
 }
+
+#[derive(Debug, Deserialize)]
+pub struct PartialProjectPathTrainResult {
+    pub departure_time: DateTime<Utc>,
+    // Ignore the rest of the payload
+}
```

Comment:
Why was this moved in `models`? And I don't understand the comment: why would a model definition care about request payloads?

Reply:
@Wadjetz up